src/System Application/App/File System/src/Account/FileAccounts.Page.al

```diff
@@ -0,0 +1,340 @@
+    SourceTable = "File Account";
+    SourceTableTemporary = true;
+    InsertAllowed = false;
+    ModifyAllowed = false;
+    DeleteAllowed = false;
+    Editable = false;
+    ShowFilter = false;
+    LinksAllowed = false;
+    RefreshOnActivate = true;
+    InherentPermissions = X;
+    InherentEntitlements = X;
+
+    layout
+    {
+        area(Content)
+        {
+            repeater(Accounts)
+            {
+                Visible = ShowLogo;
+                FreezeColumn = NameField;
+                field(LogoField; Rec.LogoBlob)
+                {
+                    ShowCaption = false;
+                    Caption = ' ';
+                    Visible = ShowLogo;
+                    ToolTip = 'Specifies the logo for the type of file account.';
+                    Width = 1;
+                }
+
+                field(NameField; Rec.Name)
+                {
+                    ToolTip = 'Specifies the name of the account.';
+                    Visible = not IsInLookupMode;
+
+                    trigger OnDrillDown()
+                    begin
+                        ShowAccountInformation();
+                    end;
+                }
+
+                field(NameFieldLookup; Rec.Name)
+                {
+                    ToolTip = 'Specifies the name of the account.';
+                    Visible = IsInLookupMode;
+                }
+
+                field(DefaultField; DefaultTxt)
+                {
+                    Caption = 'Default';
+                    ToolTip = 'Specifies whether the file account will be used for all scenarios for which an account is not specified. You must have a default file account, even if you have only one account.';
+                    Visible = not IsInLookupMode;
+                }
+
+                field(FileConnector; Rec.Connector)
+                {
+                    ToolTip = 'Specifies the type of file extension that the account is added to.';
+                    Visible = false;
+                }
+            }
+        }
+
+        area(factboxes)
+        {
+            part(Scenarios; "File Scenarios FactBox")
+            {
+                Caption = 'File Scenarios';
+                SubPageLink = "Account Id" = field("Account Id"), Connector = field(Connector), Scenario = filter(<> 0); // Do not show Default scenario
+            }
+        }
+    }
+
+    actions
+    {
+        area(Creation)
+        {
+            action(View)
+            {
+                Image = View;
+                ToolTip = 'View settings for the file account.';
+                ShortcutKey = return;
+                Visible = false;
+
+                trigger OnAction()
+                begin
+                    ShowAccountInformation();
+                end;
+            }
+
+            action(AddAccount)
+            {
+                Image = Add;
+                Caption = 'Add an file account';
+                ToolTip = 'Add an file account.';
```

Comment:
More meaningful tool tip <> Caption?

Reply:
Do you have a suggestion? I took the entire page from the email module. It's exactly the same there.

Reply:
Opens a File Account Wizard setup page in order to add an File Account.

Reply:
thx.